client.py

```diff
@@ -920,6 +934,12 @@ def __init__(self, window):
             do_markdown = True
             self.MarkdownCheck.setChecked(True)
 
+        if isinstance(do_secure, bool):
+            self.SecureCheck.setChecked(do_secure)
+        else:
+            do_secure = True
+            self.SecureCheck.setChecked(True)
```

Comment:
Change this to `self.SecureCheck.setChecked(do_secure)`, so it can be like the markdown check.

Reply:
Are you referring to changing the else condition to look like L934-L935?

Reply:
Yes. It just bugs me a little.